packages/server/lib/controllers/proxy.controller.ts

```diff
@@ -301,14 +307,14 @@ class ProxyController {
             }
 
             if (!isJsonResponse) {
-                res.send(responseData);
+                res.send(Buffer.concat(responseData));
                 await logCtx.success();
                 metrics.increment(metrics.Types.PROXY_SUCCESS);
                 return;
             }
 
             try {
-                const parsedResponse = JSON.parse(responseData);
+                const parsedResponse = JSON.parse(Buffer.concat(responseData).toString());
```

Comment:
not sure why we parse the json instead of just forwarding the bytes. maybe @khaliqgant knows

Reply:
Yeah, I figured I would come back to that next week since folks are starting to wind down, but it feels like we could just use that block on line 278 as our handling for everything and it would feasibly shut down any issues here.

Reply:
Really don’t recall. Remember going through a lot of iterations of this to handle edge cases though. Not to be alarmist but would just say to express caution with changes without proper testing. Also if it breaks we’ll probably hear about it pretty quickly since the proxy is pretty core

Reply:
It's not so long ago, but I can't really find for which integration it was. 
I would say the logging part is purely arbitrary and we could without but we'll see
https://github.com/NangoHQ/nango/pull/2228/files